js/main.js

```diff
@@ -0,0 +1,10 @@
+$("#more").click(function(){
+    $("section").css({"position": "fixed"});
+    $("#info").animate({"left": "10%",}, 650);
+    $("#info").animate({"left": "0%",}, 300);
+});
+
+$("#back").click(function(){
+	$("section").css({"position": "absolute"});
+    $("#info").animate({"left": "-110%"}, "slow");
```

Comment:
When we're rich and famous we can do AB testing and host focus groups but for now long discussions are the way to go.